agent/agentscripts/agentscripts.go

```diff
@@ -71,6 +78,18 @@ type Runner struct {
 	cron          *cron.Cron
 	initialized   atomic.Bool
 	scripts       []codersdk.WorkspaceAgentScript
+
+	// Metrics
+	// scripts includes scripts that are scheduled.
```

Comment:
nit: this comment may sound a bit blurry for the first-time reader. Do you think that we can rephrase it into something more descriptive?

Reply:
```golang
	// scriptsExecuted includes all scripts executed by the workspace agent.
	// Agents execute startup scripts, and scripts on a cron schedule. Both of
	// which will increment this counter.
	scriptsExecuted *prometheus.CounterVec
```